packages/component/src/hooks/internal/useAcknowledgedActivity.js

```diff
@@ -0,0 +1,61 @@
+import { hooks } from 'botframework-webchat-api';
+import { useMemo, useRef } from 'react';
+import { useSticky } from 'react-scroll-to-bottom';
+
+import findLastIndex from '../../Utils/findLastIndex';
+import getActivityUniqueId from '../../Utils/getActivityUniqueId';
+import useChanged from './useChanged';
+
+const { useActivities } = hooks;
+
+// Acknowledged means either:
+// 1. The user sent a message
+//    - We don't need a condition here. When Web Chat send the message, it will scroll to bottom, and it will trigger condition 2 below.
```

Comment:
```suggestion
//    - We don't need a condition here. When Web Chat sends the message, it will scroll to bottom, and it will trigger condition 2 below.
```